xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 
-<a name="roleofxapi"/>
-## 1.0 Summary of the Experience API
+The Experience API (xAPI) is a technical specification that aims to facilitate the documentation and communication of learning experiences.  It does so by setting out a protocol that specifies how learning experiences should be described and an Application Programming Interface that sets out how information about these experiences should be stored and exchanged electronically.  
 
-The Experience API (xAPI) is an Application Programming Interface (API) specification that 
-facilitates communication about experiences between a Learning Record Store and a client, 
-typically an Activity Provider. This specification was developed with learning experiences in mind, 
-but many other kinds of experiences could be tracked using xAPI. 
+The xAPI is part of the Advanced Distributed Learning (ADL) initiative, established in 1997 to standardize and modernize training and education management and delivery in the US. Among the many outcomes of this initiative was SCORM - the Shareable Content Object Reference Model. SCORM has 
+become a great success and an internationally recongized framework that has greatly contributed greatly to the economy and effectiveness of online learning.
+  
+While SCORM has been a great success, there has been a growing realization that there is a huge need and potential to track learning experiences of individuals beyond formal, structured computer-based traing.  In assessing candidates' suitability for positions or their capability form performing various tasks, there is a need to consider a wide range of formal and informal learning experiences, both on and offline.  That information, more often than not be scattered across a wide variety of sources.
 
-This specification does not attempt to standardize how the Activity Provider facilitates and tracks the learning experience, 
-nor how the Learning Store stores and procesess that data. The scope of this specification is limited to the requests and responses
-between Learning Record Store and client. 
+Out of this perceived need, the xAPI initiative and specification was born.  xAPI assumes that:
+  * There is a need to be able to analyze information about learning experiences distributed across a wide variety of sources.
+  * Developing a commonly-accepted framework for gathering, storing and exchanging this information represents the best way of achieving this.
 
-The purpose of the xAPI and the goals of this document are:
-* To maximize interoperability of systems which create, gather and store information  
-about learning experiences.
-* To provide a guide to those who want to build applications that conform to and implement this 
-specification.
+The goals of the xAPI are:
+
+* To make it easier to understand and compare learning experiences recorded in a wide variety of places.
+* To maximize interoperability of systems which create, gather, store and process information about learning experiences.
```

Comment:
gathering and storing are fine here, because we're talking about interoperability between systems that do this, not setting out how these things are done.